### ISSUE TITLE
UnhandledPromiseRejectionWarning: Error: An object could not be cloned.

### ISSUE THREAD

**Issue by AllenGu93** [electron/electron#23743]
### Preflight Checklist

* [ ✅] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [ ✅] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ ✅] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  9.0.0
* **Operating System:**
  mac 10.15.3
* **Last Known Working Electron version:**
  8.3.0

### Expected Behavior
executeJavaScript doesn't throw an error.

### Actual Behavior
$ npm start

> Tracup@2.3.5 start /Users/allengu/Desktop/tracup-client_4
> electron .

(node:2169) UnhandledPromiseRejectionWarning: Error: An object could not be cloned.
    at EventEmitter../lib/renderer/ipc-renderer-internal.ts.exports.ipcRendererInternal.send (electron/js2c/re
nderer_init.js:2324:20)
    at EventEmitter.<anonymous> (electron/js2c/renderer_init.js:2286:26)
(node:2169) UnhandledPromiseRejectionWarning: Error: An object could not be cloned.
    at EventEmitter../lib/renderer/ipc-renderer-internal.ts.exports.ipcRendererInternal.send (electron/js2c/re
nderer_init.js:2324:20)
    at EventEmitter.<anonymous> (electron/js2c/renderer_init.js:2286:26)
(node:2169) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by thr
owing inside of an async function without a catch block, or by rejecting a promise which was not handled with
.catch(). (rejection id: 1)
(node:2169) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by thr
owing inside of an async function without a catch block, or by rejecting a promise which was not handled with
.catch(). (rejection id: 1)
(node:2169) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise
rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:2169) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise
rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:2169) UnhandledPromiseRejectionWarning: Error: An object could not be cloned.
    at EventEmitter../lib/renderer/ipc-renderer-internal.ts.exports.ipcRendererInternal.send (electron/js2c/re
nderer_init.js:2324:20)

### Additional Information

upgrade the electron to the latest version, i found the error, i reviewed the document but I didn't get it, how can I solve this problem?

**Comment by codebytere** [comment]
This is expected - see https://github.com/electron/electron/blob/8de06f0c571bc24e4230063e3ef0428390df773e/docs/breaking-changes.md#L184-L233 for more details.

**Comment by yanguoyu** [comment]
There is also an `UnhandledPromiseRejectionWarning: Error: An object could not be cloned.`
I found it because I have referred to `electron-log`@4.0.0. https://github.com/megahertz/electron-log/blob/95506c0aa89b6de15aae11e7fe3873501d556533/src/electronApi.js#L122
From this comment https://github.com/electron/electron/issues/23722#issuecomment-632631774, if the first parameter of `executeJavaScript` does not end of `;0`, the resulting value is attempted to be cloned and used as a result of `executeJavaScript`. So the `electron-log@4.0.0` will throw an unhandled promise reject, and it has been fixed in the latest version.
